website/content/docs/getting-started/deploy-and-login.mdx

```diff
@@ -0,0 +1,198 @@
+---
+layout: docs
+page_title: Deploy and Login
+description: |-
+  How to deploy HCP Boundary services and login for the first time.
+---
+
+# Deploy an HCP Boundary Instance
+
+Navigate to HashiCorp Cloud Platform and login using your credentials. Click on
+the **Boundary (Beta)** tab and click **Deploy Boundary** on the right-hand
+pane. 
+
+Fill out the following form details:
+
+- Instance Name
+- Administrator Username
+- Administrator Password
+
+Store the administrator username and password in a safe location. These
+credentials will be used when authenticating using the Admin Console, CLI, or
+Terraform.
+
+Click **Deploy** after completing the form. It may take a few minutes to
+initialize the HCP Boundary instance.
+
+![Deploy boundary](/img/hcp-deploy-form.png)
+
+## Login to HCP Boundary
+
+Once the deployment is complete, the HCP Boundary instance can be accessed
+directly from the HCP Boundary portal using the credentials created during
+setup. 
+
+From the HCP Boundary portal page, click the **Open Admin UI** button. 
+
+Enter the admin username and password you created when you deployed the new
+instance and click **Authenticate**.
+
+The **Welcome to Boundary** setup wizard will launch automatically. There are
+two options for Boundary’s initial setup:
+
+1. Quick setup
+1. Manual
+
+![Setup Wizard](/img/setup-wizard.png)
+
+The **Quick setup** is recommended for new users, and generates a set of initial
+resources to assist in connecting to your first target. 
+
+![Setup Wizard](/img/setup-wizard-target.png)
+
+The **Manual** setup drops the user directly into Boundary, without any
+provisioned resources.
+
+Upon completion of the setup wizard you are logged into your HCP Boundary
+instance's Global scope via the web UI. This is the default scope for all new
+Boundary clusters.
+
+Click **Auth Methods**.
+
+Click the **copy icon** for the Password **Auth Method ID**. The ID is needed to
+authenticate to HCP Boundary using the CLI.
+
+<Tabs>
+<Tab heading="CLI">
+
+
+In the HCP Boundary portal, open the **Boundary Overview** page and click the
+**copy icon** in the **Copy this into Boundary Desktop** section. This copies
+your environment’s Boundary **Origin URL**.
+
+Open a terminal session and set the `BOUNDARY_ADDR` environment variable to the
+copied origin URL.
+
+```shell-session
+$ export BOUNDARY_ADDR=<boundary-origin-url>
+```
+
+Log in with the administrator credentials you created when you deployed the HCP
+Boundary instance.
+
+```shell-session
+$ boundary authenticate password \
+   -auth-method-id=COPIED_AUTH_ID \
+   -login-name=ADMIN_USERNAME \
+   -password=ADMIN_PASSWORD
+```
+
+You are now logged into your HCP Boundary instance's Global scope via the CLI.
+This is the default scope for all new Boundary clusters.
+
+If you are on Unix-like operating system (other than macOS/Darwin), you may get
+an error indicating that the token could not be stored, as the freedesktop.org
```

Comment:
DBUS-based secret service stuff is pretty problematic and very often doesn't work in terminal sessions. I recommend it only for those that know what they're doing. The default on *nix systems is `pass` and is the recommended approach.

Reply:
I'd appreciate if you could author an alternative bit of text if you want this updated, since it's copied directly from the existing [run and login docs](https://www.boundaryproject.io/docs/getting-started/run-and-login#login-to-boundary).

Reply:
I'm not sure how you'd want to construct this. Would you want to point people to pass? Do you want to include any setup or configuration information for it? Would you want any other text as a backup, or to mention that alternatives are available?